### PR TITLE
Updated Deploying Certificates topic to address Jira issue NGDOC-49

### PR DESCRIPTION
**Pull request by marcfishelquantil**

### CODE REVIEW

**Comment by burtonquantil** [comment]
@marcfishelquantil  the pull request title refers to NGDOC-49, but I do not see how the change in this pull request is related.

**Comment by marcfishelquantil** [comment]
@burtonquantil Please ignore reference to NGDOC-49 in subject line. Fix was to change the width of the figure.

**Comment by burtonquantil** [comment]
What's happening with this pull request? Are you still working on changes?

**Comment by marcfishelquantil** [comment]
I just corrected typos in cdn/apispec/openapi.json. Please review.

**Comment by marcfishelquantil** [comment]
On Friday, I updated the Edge Logic online help. Please review it.

**Comment by burtonquantil** [comment]
@marcfishelquantil  You need to resolve the conflict in this pull request.

File: docs/edge-logic/intro.md [hunk 1]
```diff
@@ -2,8 +2,8 @@
...
 
-Starting from 2016, we made significant changes to the open-source version of the software to enhance it as a CDN proxy server. One of CDN360’s main design goals is to give users as much control as possible to self-provision the service. Based on this rationale, we decided to allow users to code in the NGINX language directly to control the proxy servers. We believe this gives them the maximum flexibility to achieve their business requirements.
+Starting from 2016, CDNetworks has made significant changes to the open-source version of the software to enhance it as a CDN proxy server. One of CDN360’s main design goals is to give you as much control as possible to self-provision the service. Based on this commitment, we allow you to code directly in the NGINX language to control the proxy servers. We believe this gives you the maximum flexibility to achieve their business requirements.
```
Comment (burtonquantil) [inline docs/edge-logic/intro.md:5]: @marcfishelquantil  you began using references to "you" but still refer to "their". That is inconsistent.

File: docs/edge-logic/staging-environment.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
+The staging environment has a few isolated CDN360 proxy servers on which you can test the property configuration. You can fully test the server’s behavior and make sure it meets your complete expectations before you deploy the property to the production servers. The staging environment serves as a run-time test environment for the code you write. For your reference, here is a workflow of the [CDN360 service provisioning](</docs/getting-started.md#quick-start>) and [how to test with the staging environment](</docs/portal/edge-configurations/testing-property.md#testing-property-in-staging>). You can also purge the staging servers to test the purging behavior. To find the IP addresses of the staging servers, run "dig staging.qtlgslb.com" in a bash console.
```
Comment (burtonquantil) [inline docs/edge-logic/staging-environment.md:5]: We refer to "servers" and then switch to singular  "server's".  Is this correct?

File: docs/edge-logic/intro.md (hunk 1 above)
Comment (marcfishelquantil) [reply docs/edge-logic/intro.md]: I have changed "their" to "your."

File: docs/edge-logic/staging-environment.md (hunk 2 above)
Comment (marcfishelquantil) [reply docs/edge-logic/staging-environment.md:5]: This is fine; however, to side-step any ambiguity, I changed the article THE to A.
Before: "You can fully test the server’s behavior and make sure it meets your complete expectations before you deploy the property to the production servers." 
After: "You can fully test a server’s behavior and make sure it meets your complete expectations before you deploy the property to the production servers."

**Pull request merged**